provider/static/static.go

```diff
@@ -104,7 +114,10 @@ provider:
     workerGroup: ...
     workerID: ...
     staticSecret: ... # shared secret configured for this worker in worker-manager
-    # custom properties for TASKCLUSTER_WORKER_LOCATION
+	# (optional) custom provider-metadata entries to be passed to worker
+	providerMetadata: {prop: val, ..}
+    # (optional) custom properties for TASKCLUSTER_WORKER_LOCATION
+	# (values must be strings)
```

Comment:
and this one

Reply:
Why?  They must be strings, it'd be rude not to document that..